### PR TITLE
Streamline contract generation

### PR DESCRIPTION
**Pull request by YaroShkvorets**
Fixed FIXME

### CODE REVIEW

File: abigen/src/build.rs [hunk 1]
```diff
@@ -21,34 +20,14 @@ impl Abigen {
...
-        //
-        // There is probably a way to avoid that somehow?
-        let file = quote! {
-            mod __remove__ {
-                #item
-            }
-        };
+        let tokens = generate_abi_code(self.abi_path.to_string_lossy()).context("generating abi code")?;
 
-        let file = syn::File {
```
Comment (maoueh) [inline abigen/src/build.rs:65]: Nice thank you! I was wondering how to do this :)

**Review by maoueh — APPROVED** [review]
Wow this one lingers so long!

**Comment by maoueh** [comment]
If you can rebase, I will merge it

**Comment by YaroShkvorets** [comment]
Rebased

**Pull request merged**